app/src/main/java/com/codepath/selfiespot/activities/ProfileActivity.java

```diff
@@ -32,11 +32,11 @@
     @BindView(R.id.toolbar)
     Toolbar mToolbar;
 
```

Comment:
i meant the java variables, the xml attributes can still have underscores, regardless ok for now